### ISSUE TITLE
Changelogs are hard to follow

### ISSUE THREAD

**Issue by kaelig** [apollographql/apollo-client#3153]
Thank you for keeping all of these packages alive!

One of my projects uses https://greenkeeper.io to keep our dependencies up to date, and every time I try to check what's in a new version of one of the apollo-* packages, I have trouble finding it.

e.g. Greenkeeper opens a PR, like so:

![screenshot 2018-03-13 12 26 30](https://user-images.githubusercontent.com/85783/37365146-dfda82bc-26b9-11e8-9e9f-df9d039b0d84.png)

Then I look in [the changelog for that package](https://github.com/apollographql/apollo-client/blob/master/packages/apollo-client/CHANGELOG.md).

![screenshot 2018-03-13 12 26 39](https://user-images.githubusercontent.com/85783/37365140-dc6407fc-26b9-11e8-944c-59a5e9587383.png)

But I can't figure out what the changes are for v2.2.7. This is the same thing for pretty much every new version of apollo-* packages.

1. Am I reading these changelogs incorrectly? Should I ignore new versions without a proper changelog?
2. Can you also publish `CHANGELOG.md` files with every package on npm? This could potentially help greenkeeper surface changes in the PRs.

**Comment by hwillson** [comment]
Thanks for the feedback @kaelig. This is a known issue, and is something that we're putting more attention towards resolving. We're working on better repo workflows/policies internally, and making sure up to date (and accurate) change logs are in place is definitely a high priority. I'll close this issue for now since this work won't be tracked here, but improvements are definitely coming. Thanks again!
